src/pages/home/components/banner-section.jsx

```diff
@@ -0,0 +1,12 @@
+export const HeroBanner = ({styles}) => {
+    return(
+        <section className={`${styles.hero} flex-col justify-c-center`} style={{backgroundImage:"url(/assets/hero-banner-2.webp),var(--bg-gradient-overlay)"}}>
```

Comment:
Inline Styles ?

Reply:
Actually, I wasn't able to access that image from the CSS file, so only way was to link in the js file